therapy/therapy-about.html

```diff
@@ -0,0 +1,160 @@
+            </div>
+            <nav class="navbar navbar-expand-sm bg-custom open-sidenav" id="open">
+                <ul class="navbar-nav">
+                    <li class="text-right close-nav">
+                        <a href="#" class="close-div nav-link text-right open-close" onclick="closeNav()">&times;</a>
+                    </li>
+                    <li class="nav-item active overlay">
+                        <a href="therapy.html" class="nav-link">Home</a>
+                    </li>
+                    <div class="divider"></div>
+                    <li class="nav-item">
+                        <a href="therapy-about.html" class="nav-link">About</a>
+                    </li>
+                    <div class="divider"></div>
+                    <li class="nav-item">
+                        <a href="therapy-services.html" class="nav-link">Services</a>
+                    </li>
+                    <div class="divider"></div>
+                    <li class="nav-item">
+                        <a href="therapy-blog.html" class="nav-link">Blog</a>
+                    </li>
+                    <div class="divider"></div>
+                    <li class="nav-item">
+                        <a href="therapy-contact.html" class="nav-link">Contact</a>
+                    </li>
+                </ul>
+            </nav>
+            <div>
+                <h3 class="headings headings-therapy"><a href="therapy.html">Maria Hope, Ph.D. - Psychologist</a></h3>
+            </div>
+        </div>
+    </header>
+    <section class="container">
+        <div class="row">
+            <div class="about">
+                <h3 class="headings about-text about-heading">About</h3>
+                <h1 class="title about-text">Maria Hope</h1>
+                <p class="header-p about-text about-p">
+                    I'm a paragraph. Click here to edit and add your own text. Is easy! Just click on "Edit Text" or double-click on me and
+                    you can add your own content and change fonts. Feel free to drag and drop anywhere on your page. I'm a great place for
+                    you to tell your story and let your customers know a little more about you. This is a great place to write a long text
+                    about your company and its services. You can use this space to go into details about your business. Talk about your team
+                    and about the services you provide. Tell your visitors about how you came up with the idea of starting your business and
+                    what makes it different from your competitors. Make your business stand out and show who you are.
+                </p>
+                <img class="col-12 about-maria" src="https://static.wixstatic.com/media/9c608a_2a78cf5937414d0e9a597504d68daafc~mv2_d_6046_4035_s_4_2.jpg/v1/fill/w_365,h_446,al_c,q_80,usm_0.66_1.00_0.01/9c608a_2a78cf5937414d0e9a597504d68daafc~mv2_d_6046_4035_s_4_2.webp" alt="maria-pic">
```

Comment:
If you look at the CSS behind Bootstrap classes, you'll see that 'row' has 'display: flex', which it uses to control the layout of its flex children: the 'col-...' classes.
Therefore, for Bootstrap to work, any 'col-' class must be applied to a **direct** child of 'row'.
In this case, the direct children of the row on line 60 are the 'about' div on line 61 and the 'experience' div below - so they should have a 'col-...' class applied to them, not the img.

Reply:
For mobile, using 'col-12' is common as most things are full width. But what about for tablet or desktop? If the content is centered but less wide, you could use 'col-md-10' plus add an offset which will move it by 1 column to ensure it's centered: 'offset-md-1'

So in this example, you would apply all the following classes to the 'about' div: 'about col-12 col-md-10 offset-md-1'